pyensign/stream.py

```diff
@@ -22,17 +23,20 @@ class StreamHandler:
 
     def __init__(
         self,
-        stub,
+        client,
```

Comment:
I like this approach as it requires fewer arguments to be passed in.  Potentially there might be more attributes that might be needed from the client in the future.

Reply:
Yeah, although it implies that there might be too much information sharing between the `Client` and `StreamHandler`, so this is probably a good candidate for refactoring later.